### PR TITLE
Update OpenUV

### PR DESCRIPTION
**Pull request by johanvandyck**
Update OpenUV (Xorfor/Domoticz-OpenUV-Plugin)

### CODE REVIEW

**Comment by johanvandyck** [comment]
Added OpenUV (Xorfor/Domoticz-OpenUV-Plugin)

**Comment by johanvandyck** [comment]
Hi
I can't see any difference, but the Domoticz log shows "Apr 28 11:59:08 localhost domoticz: (Python Plugin Manager) Error Line '#011    "OpenUV":#011#011#011    ["Xorfor",#011#011"Domoticz-OpenUV-Plugin,#011#011"Open UV",#011#011#011     "master"],#012'"
I seems some invisible chars are used in my file. Or other error.

**Comment by ycahome** [comment]
Am not sure that am understanding your Pull request changes.
Did you removed "Pi-hole summary" and added "OpenUV"?If yes, why you left it on the second table?

Pls justify

**Comment by johanvandyck** [comment]
Hi

No, I'm trying to add "OpenUV". I'm not removing "Pi-hole summary" or anything else.

File: plugin.py [hunk 1]
```diff
@@ -55,7 +55,8 @@
                 <option label="MQTT discovery" value="MQTTDiscovery"/>
                 <option label="Onkyo AV Receiver" value="Onkyo"/>
                 <option label="OpenAQ" value="xfr_openaq"/>
-                <option label="Pi-hole summary" value="xfr_pihole"/>
```
Comment (ycahome) [inline plugin.py]: Your "commit" says that "Pi_hole summary" will be removed.  Sorry, cannot merge it like that.
Please correct your code or delete your request and repost it!

regards,

**Pull request closed without merging**